### ISSUE TITLE
Matsim built with Java 11 but does not say to require it

### ISSUE THREAD

**Issue by don-vip** [matsim-org/josm-matsim-plugin#113]
Latest version of plugin has been built with Java 11:

```
matsim.jar;https://github.com/matsim-org/josm-matsim-plugin/releases/download/v1.0.9/matsim.jar
	Manifest-Version: 1.0
	Created-By: 11.0.4 (JetBrains s.r.o)
	Gradle-Version: 6.8.3
	Groovy-Version: 2.5.12
	Plugin-Date: 2021-02-27T02:09:14+0100
```

JOSM is still being used by many Java 8 users, so please rebuild it with Java 8.

If that's not possible because you need Java 11, please add `Plugin-Minimum-Java-Version: 11` in the manifest, so that JOSM can skip plugin loading with Java 8 (but that means a lot of users cannot use your plugin anymore).

See https://josm.openstreetmap.de/ticket/20547 for reference.

**Comment by nkuehnel** [comment]
Hi,

I see. I will try to rebuild with Java 8, thanks

**Comment by nkuehnel** [comment]
fixed in 9595d17fe56883ad0362d29163f1902eeb709b82
